src/Files.App/Dialogs/CreateArchiveDialog.xaml

```diff
@@ -0,0 +1,164 @@
+
+			<Style x:Key="GridSettingStyle" TargetType="Grid">
+				<Setter Property="Background" Value="{ThemeResource ExpanderHeaderBackground}" />
+				<Setter Property="BorderBrush" Value="{ThemeResource ExpanderHeaderBorderBrush}" />
+				<Setter Property="BorderThickness" Value="{ThemeResource ExpanderHeaderBorderThickness}" />
+				<Setter Property="HorizontalAlignment" Value="Stretch" />
+				<Setter Property="Padding" Value="-24,4,20,4" />
+				<Setter Property="MinHeight" Value="{StaticResource ExpanderMinHeight}" />
+				<Setter Property="CornerRadius" Value="{ThemeResource ControlCornerRadius}" />
+			</Style>
+
+			<ResourceDictionary.MergedDictionaries>
+				<ResourceDictionary Source="/ResourceDictionaries/RightAlignedToggleSwitchStyle.xaml" />
+			</ResourceDictionary.MergedDictionaries>
+		</ResourceDictionary>
+	</ContentDialog.Resources>
+
+	<StackPanel Padding="0,8,0,0" Spacing="24">
+
+		<Grid ColumnSpacing="8">
+			<Grid.ColumnDefinitions>
+				<ColumnDefinition Width="*" />
+				<ColumnDefinition Width="Auto" />
+			</Grid.ColumnDefinitions>
+
+			<TextBox
+				x:Name="FileNameBox"
+				Width="340"
+				VerticalAlignment="Bottom"
+				Header="{helpers:ResourceString Name=ArchiveName}"
+				PlaceholderText="{helpers:ResourceString Name=EnterName}"
+				Text="{x:Bind FileName, Mode=TwoWay}" />
+			<ComboBox
+				x:Name="FileFormatSelector"
+				Grid.Column="1"
+				MinWidth="100"
+				VerticalAlignment="Bottom"
+				ItemsSource="{x:Bind FileFormats}"
+				SelectedValue="{x:Bind FileFormat, Mode=TwoWay}"
+				SelectedValuePath="Key"
+				SelectionChanged="FileFormatSelector_SelectionChanged">
+				<ComboBox.ItemTemplate>
+					<DataTemplate>
+						<Grid ColumnSpacing="24">
+							<Grid.ColumnDefinitions>
+								<ColumnDefinition Width="Auto" />
+								<ColumnDefinition Width="*" />
+							</Grid.ColumnDefinitions>
+							<TextBlock Text="{Binding Label}" />
+							<TextBlock
+								Grid.Column="1"
+								VerticalAlignment="Bottom"
+								FontSize="12"
+								FontWeight="Light"
+								Text="{Binding Description}"
+								TextWrapping="Wrap"
+								Visibility="{Binding IsDropDownOpen, ElementName=FileFormatSelector}" />
+						</Grid>
+					</DataTemplate>
+				</ComboBox.ItemTemplate>
+			</ComboBox>
+		</Grid>
+
+		<StackPanel Spacing="4">
+			<Grid Style="{StaticResource GridSettingStyle}">
+				<settings:SettingsDisplayControl Title="{helpers:ResourceString Name=CompressionLevel}" HorizontalAlignment="Stretch">
+					<ComboBox
+						x:Name="CompressionLevelSelector"
+						HorizontalAlignment="Right"
+						DisplayMemberPath="Label"
+						ItemsSource="{x:Bind CompressionLevels}"
+						SelectedValue="{x:Bind CompressionLevel, Mode=TwoWay}"
+						SelectedValuePath="Key" />
+				</settings:SettingsDisplayControl>
+			</Grid>
+			<Grid
+				x:Name="SplittingSizeLayout"
+				Style="{StaticResource GridSettingStyle}"
+				Visibility="Collapsed">
```

Comment:
Let's switch to `x:Load` and bind to a property in the ViewModel.

Reply:
It's done